docs/intro/whatsnext.txt

```diff
@@ -123,8 +123,8 @@ ticket system and use your feedback to improve the documentation for everybody.
 
 Note, however, that tickets should explicitly relate to the documentation,
 rather than asking broad tech-support questions. If you need help with your
-particular Django setup, try the |django-users| mailing list or the `#django
-IRC channel`_ instead.
+particular Django setup, try the Django Community page at https://www.djangoproject.com/community
```

Comment:
Here and for `docs/ref/contrib/index.txt`: we should use an actual reStructuredText link, not just the URL within text.